iOS Example/iOS Example.xcodeproj/project.pbxproj

```diff
@@ -252,8 +253,10 @@
 				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
 				CLANG_WARN_EMPTY_BODY = YES;
 				CLANG_WARN_ENUM_CONVERSION = YES;
+				CLANG_WARN_INFINITE_RECURSION = YES;
```

Comment:
@NachoSoto I think I've done this correctly - mind taking another look at Common/Release/Debug xcconfig?